packages/cli/test/unit/index.test.ts

```diff
@@ -140,6 +142,28 @@ describe('main', () => {
       });
     });
 
+    describe('tracking enabled', () => {
+      it('is false when VERCEL_TELEMETRY_DISABLED set', () => {
+        const configThatWillBeIgnoredAnyway = {
+          enabled: true,
+        };
+
+        vi.stubEnv('VERCEL_TELEMETRY_DISABLED', '1');
+        const output = new Output(process.stderr, {
+          debug: true,
+          noColor: false,
+        });
+
+        const telemetryEventStore = new TelemetryEventStore({
+          isDebug: true,
+          output,
+          config: configThatWillBeIgnoredAnyway,
+        });
+
+        expect(telemetryEventStore.enabled).toBe(false);
```

Comment:
```suggestion
        expect(telemetryEventStore.enabled()).toBe(false);
```

Reply:
Oh, weird, I assumed we'd make that a getter. Going to make it getter!